capa/main.py

```diff
@@ -599,6 +558,9 @@ def handle_common_args(args):
 
 
 def main(argv=None):
+    if sys.version_info < (3, 6):
+        raise UnsupportedRuntimeError("capa should only be used with Python 3.6+")
```

Comment:
nice! should this be caught as well though?

Reply:
The idea is that capa raises an exception and stop executing if the wrong python version is used (as it may not run properly in that case). Or do you mean something else?

Reply:
That's good. Please disregard my previous comment.